modules/custom/az_paragraphs/src/Plugin/migrate/process/ParagraphsBehavior.php

```diff
@@ -59,21 +63,24 @@ public function transform($value, MigrateExecutableInterface $migrate_executable
         'bg-trans-sky' => 'light',
         'bg-trans-arizona-blue' => 'dark',
         'bg-trans-black' => 'dark',
+        'dark' => 'dark',
+        'light' => 'light',
+
       ];
       foreach ($bg_text_color as $bg_text_color_item) {
-        $behavior['bg_color'] = $bg_text_color_mapping[$bg_text_color_item['value']];
+        $behavior['az_text_media_paragraph_behavior']['bg_color'] = $bg_text_color_mapping[$bg_text_color_item['value']];
       }
     }
 
     // Background Attachment.
     if (!empty($this->configuration['bg_attach'])) {
-      $bg_attach = $row->getSourceProperty($this->configuration['bg_attach']);
-      $bg_attach_mapping = [
-        'bg-attachment-fixed' => 'bg-fixed',
-        'bg-attachment-scroll' => '',
-      ];
-      foreach ($bg_attach as $bg_attach_item) {
-        $behavior['bg_attachment'] = $bg_attach_mapping[$bg_attach_item['value']];
+      if (!empty($row->getSourceProperty($this->configuration['bg_attach']))) {
+        $bg_attach_mapping = [
+          'bg-attachment-fixed' => 'bg-fixed',
+          'bg-fixed' => 'bg-fixed',
```

Comment:
I'm wondering why these 2 different `bg-fixed` source values are needed here.  Do the values in the demo content JSON files not match the D7 Quickstart value?

Reply:
> I'm wondering why these 2 different `bg-fixed` source values are needed here. Do the values in the demo content JSON files not match the D7 Quickstart value?

Never mind.  I think I see what's going on now.